scripts/bridgeERC20.ts

```diff
@@ -197,76 +109,57 @@ const FUEL_FUNGIBLE_TOKEN_ADDRESS = process.env.FUEL_FUNGIBLE_TOKEN_ADDRESS || '
   console.log('Tokens were bridged to Fuel successfully!!');
 
   // note balances of both accounts after transfer
-  console.log('Account balances:');
-  console.log(
-    `  Ethereum - ${ethers_formatToken(await ethTestToken.balanceOf(ethAccountAddr))} Tokens (${ethAccountAddr})`
-  );
-  console.log(
-    `  Fuel - ${fuels_formatToken(await fuelAccount.getBalance(fuelTestTokenId))} Tokens (${fuelAccountAddr})`
-  );
-  console.log('');
+  await logTokenBalances(ethTestToken, ethAcct, fuelAcct, fuelTestTokenId);
 
   /////////////////////////////
   // Bridge Fuel -> Ethereum //
   /////////////////////////////
 
   // withdraw tokens back to the base chain
   console.log(`Sending ${TOKEN_AMOUNT} Tokens from Fuel...`);
-  const paddedAddress = '0x' + ethAccountAddr.slice(2).padStart(64, '0');
+  const paddedAddress = '0x' + ethAcctAddr.slice(2).padStart(64, '0');
   const scope = fuelTestToken.functions
     .withdraw(paddedAddress)
     .callParams({
       forward: { amount: fuels_parseToken(TOKEN_AMOUNT, 9), assetId: fuelTestTokenId },
     })
-    .txParams(fuelTxParams);
-  scope.transactionRequest.addMessageOutputs(1);
+    .txParams(FUEL_TX_PARAMS);
+  // scope.transactionRequest.addMessageOutputs(1);
```

Comment:
Why has this been commented out?

Reply:
This type of output doesn't exist on the specs and the SDK. And the SDK takes care of adding all missing outputs like output variable.

Reply:
I think this commented out line should be removed if it's not needed

Reply:
Done